diracx-client/tests/test_regenerate.py

```diff
@@ -64,6 +63,18 @@ def test_regenerate_client(test_client, tmp_path):
 
     subprocess.run(cmd, check=True)
 
+    # cmd = [
+    #     "autorest",
+    #     "--postprocess",
+    #     "--python",
+    #     f"--input-file={openapi_spec}",
+    #     "--models-mode=msrest",
+    #     "--namespace=generated",
+    #     f"--output-folder={output_folder.parent}",
+    # ]
+
+    # subprocess.run(cmd, check=True)
+
```

Comment:
Move to an issue if we don't already have one
```suggestion
```

Reply:
We do :-) https://github.com/DIRACGrid/diracx/issues/302